Remotion/Web/Contracts.DiagnosticMetadata/DiagnosticMetadataAttributes.cs

```diff
@@ -34,6 +34,8 @@ public static class DiagnosticMetadataAttributes
     public static readonly string TriggersPostBack = "data-triggers-postback";
 
 
+    public static readonly string WebButtonButtonType = "data-webbutton-button-type";
+
```

Comment:
```suggestion
```suggestion
    public static readonly string ButtonType = "data-button-type";


```
We can simplify this.